testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -821,3 +821,6 @@ Feature: Synchronize products in the products page of the Setup Wizard
     And I wait until I do not see "Loading" text
     Then I should not see a "Operation not successful" text
     And I should not see a warning nor an error sign
+
+  Scenario: Detect repository synchronization issues in reposync logs
+    Then the reposync logs should not report errors
```

Comment:
I think scenarios should start with When no?
```suggestion
    When the reposync logs should not report errors
```

Reply:
This is a post-condition so 'Then', in my opinion.

Reply:
Yes, everything with "should" should be "Then".

"Then it should be so..."